packages/modeling/src/geometries/path2/concat.js

```diff
@@ -24,7 +25,9 @@ const concat = (...paths) => {
   }
   let newpoints = []
   paths.forEach((path) => {
-    newpoints = newpoints.concat(toPoints(path))
+    const tmp = toPoints(path)
+    if (newpoints.length > 0 && equals(tmp[0], newpoints[newpoints.length - 1])) tmp.shift()
```

Comment:
It's possible to concat paths that are empty, i.e. no points. So an addition check is required. Maybe the new test case should include an empty path.